src/lib/containers/download_list_v2/DownloadCartPage.tsx

```diff
@@ -135,7 +135,7 @@ export const DownloadCartPage:React.FunctionComponent<DownloadCartPageProps> = (
                   setIsShowingDownloadSuccessAlert(true)
                   refetch()
```

Comment:
This refetch is really important -- could be worth adding a comment like 
`// we refetch the data because the backend will instantly remove the downloadable files from the download list after a package has been created` (assuming my understanding of the feature is correct 🙂)